api/src/test/java/marquez/db/LineageDaoTest.java

```diff
@@ -888,4 +889,72 @@ public void testGetCurrentRunsGetsLatestRun() {
         .extracting(r -> r.getId().getValue())
         .containsAll(expectedRunIds);
   }
+
+  @Test
+  public void testGetRunLineage() {
+
+    UpdateLineageRow writeJob =
+        LineageTestUtils.createLineageRow(
+            openLineageDao,
+            "writeJob",
+            "COMPLETE",
+            jobFacet,
+            Arrays.asList(),
+            Arrays.asList(dataset));
```

Comment:
Would it make sense to add input dataset here read from some other job? 
Currently recursion is not tested.

Reply:
done